client/settings.html

```diff
@@ -68,14 +70,27 @@ <h3>Settings</h3>
             <span class="label-body">Automatically download new episodes to the disk</span>
         </label>
      
-        <label for="appendDateToFileName">
-            <input type="checkbox" name="appendDateToFileName" v-model="appendDateToFileName">
-            <span class="label-body">Append episode date to episode file name</span>
-        </label>
-       
-        <label for="appendEpisodeNumberToFileName">
-            <input type="checkbox" name="appendEpisodeNumberToFileName" v-model="appendEpisodeNumberToFileName">
-            <span class="label-body">Append episode number to episode file name</span>
+        <label for="fileNameFormat">
+            <span class="label-body">Episode file name format:</span>
```

Comment:
This is the text field where the format is entered. Ideally this would be less direct and populated by dragging tokens in to the field or by selection from a menu, but it's a start at least.

Reply:
I added a bit of Javascript to help populate the format field when the format keys are clicked.